### ISSUE TITLE
Popup-Notification about Security Risk

### ISSUE THREAD

**Issue by MaximilianBecker** [PeterCxy/Shelter#187]
Halloe everyone,
I use Shelter to isolate Whatsapp. Today i received a pop up message with the following content:

"Security Risk
Shelter seems to be infected, immediat uninstall recommendet"

At the botton of the message was a uninstall button. Has anyone ever encoutered such event and give me a hint what to do in such an event.

As a measure of precaution I uninstalled shelter and deleted the work profile it created (which uninstalled all apps in it). 

![photo_2021-02-11_09-08-11](https://user-images.githubusercontent.com/45111996/107613667-15dab800-6c49-11eb-80e2-8ad5b51bf69a.jpg)

**Comment by LinuxOpa** [comment]
see #188

**Comment by citydelver** [comment]
I received it as well.

And then I accidently uninstalled it. Accidently because I decided to think of it as a false positive (virus total only shows detection for 1 among 60 antivirus engines, see issue already referenced above). That doesn't exclude the work profile nor the apps already installed in it. But I wanted to be able to keep managing it if necessary. So I reinstalled it from the Google Play store. The issue is: it (Shelter) recognizes the existing work profile, yet it doesn't recognize itself as its managing app:

![Screenshot_20210308_110305_com.android.managedprovisioning.jpg](https://user-images.githubusercontent.com/1512515/110307721-00ee0c00-7fde-11eb-9544-62bc685fbaf1.jpg)

Can't it "identify itself" as the same app previously installed?

Should I create a new issue about this or can we concentrate this discussion over here?
